db.go

```diff
@@ -502,6 +504,17 @@ func SetOpentracingEnabled(enabled bool) DBOption {
 	}
 }
 
+// SetStartSpanOption will apply the span option function to every opentracing span created by squalor
+func SetStartSpanOption(spanOption opentracing.StartSpanOption) DBOption {
+	return func(db *DB) error {
+		if db.SpanOptions == nil {
```

Comment:
You don't need the `if` since `append` will create the slice if needed

Reply:
You could also make this function take varargs `...opentracing.StartSpanOption` and append them all

Reply:
Good idea - fixed